lib/api/model/model.dart

```diff
@@ -275,80 +286,75 @@ class ZulipStream {
   Map<String, dynamic> toJson() => _$ZulipStreamToJson(this);
 }
 
+/// Policy for which users can post to the stream.
+///
+/// For docs, search for "stream_post_policy"
+/// in <https://zulip.com/api/get-stream-by-id>
+@JsonEnum(valueField: 'apiValue')
+enum StreamPostPolicy{
+  any(apiValue: 1),
+  administrators(apiValue: 2),
+  fullMembers(apiValue: 3),
+  moderators(apiValue: 4),
+  unknown(apiValue: null);
+
+  const StreamPostPolicy({
+    required this.apiValue,
+  });
+
+  final int? apiValue;
+
+  int? toJson() => apiValue;
+}
+
 /// As in `subscriptions` in the initial snapshot.
 ///
 /// For docs, search for "subscriptions:"
 /// in <https://zulip.com/api/register-queue>.
 @JsonSerializable(fieldRename: FieldRename.snake)
-class Subscription {
-  // First, fields that are about the stream and not the user's relation to it.
-  // These are largely the same as in [ZulipStream].
-
-  final int streamId;
-  final String name;
-  final String description;
-  final String renderedDescription;
-
-  final int dateCreated;
-  final int? firstMessageId;
-  final int? streamWeeklyTraffic;
-
-  final bool inviteOnly;
-  final bool? isWebPublic; // TODO(server-??): doc doesn't say when added
-  final bool historyPublicToSubscribers;
-  final int? messageRetentionDays;
+class Subscription extends ZulipStream {
```

Comment:
api: Change Subscription to extend from ZulipStream

This can now say `api [nfc]:`, which is nice.